packages/jest-expo/src/preset/expoModules.js

```diff
@@ -92,10 +97,6 @@ module.exports = {
     scheduleNotificationWithTimer: { type: 'function', functionType: 'promise' },
     setBadgeNumberAsync: { type: 'function', functionType: 'promise' },
   },
-  ExpoRandom: {
-    getRandomBase64StringAsync: { type: 'function', functionType: 'promise' },
-    getRandomBase64String: { type: 'function' },
-  },
   ExponentScopedModuleRegistry: {},
   ExponentSplashScreen: {
```

Comment:
This shouldn't be there 😉

Reply:
It looks like `EXSplashScreen.m` hasn't been removed 😕